process_schemaless.py

```diff
@@ -102,15 +102,45 @@ def store_seen_id(row, header, seen_set):
 ])
 
 
-# TODO data freshness table, which is not on a per-project basis
+_FIELDS_TO_CHECK = set()
+_FIELDS_TO_CHECK.add('date_opened')
+_FIELDS_TO_CHECK.add('date_closed')
```

Comment:
Just for my own understanding, aren't these fields source-dependent? Why do we lookf or this field in each source, isn't it only pertinent to ppts for example?

Reply:
yep you're right.